src/ResourceManager/OperationalInsights/Commands.OperationalInsights/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.OperationalInsights" version="0.18.0-preview" targetFramework="net45" />
-</packages>
+  <package id="Microsoft.Azure.OperationalInsights" version="0.9.0-preview" targetFramework="net452" />
```

Comment:
Note - you will need to add this assembly to the psd1 file for your module in the 'RequiredAssemblies' property